text/0161-telemetry-schema-and-resource.md

```diff
@@ -0,0 +1,196 @@
+# Telemetry Schema and Resource
+
+Specify a mechnism for users of Resource to interact with Resource leveraging
+TelemetrySchema version for a stable API.
+
+## Motivation
+
+In the [current definition](https://github.com/open-telemetry/opentelemetry-specification/pull/1692)
+of Resource + Telemetry schema interaction, the specification focuses mostly on
+how to provide a consistent "storage" of Resource within the SDK. However,
+from a usage standpoint, how do consumers gain stability across changes to
```

Comment:
Who is a "consumer" here? End-user or backend receiving telemetry or SDK component, such as exporter?

Reply:
SDK component (e.g. exporter)